extras/nightly-container/build.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
```

Comment:
Can we put a `set -e' at the top, so that the script fails in case of any errors
than continuing?

We do not do any error checks, so set -e should help in exiting as soon as some
error condition is hit.
For example for some reason if provision.yml is not present or errors out...

Reply:
Will do.